sys/cheri/revoke.h

```diff
@@ -96,16 +101,16 @@ cheri_revoke_is_revoked(const void * __capability cap)
  *
  * In particular, we used to define the shadow as the (padded) concatenation of
  * four structures: the fine-grained memory shadow, the coarse-grained memory
- * shadow, the otype shadow, and the info page.  Unpleasantly, we did so
- * identically for all supporting architectures.  Moreover, while this worked
+ * shadow, the otype shadow, and the info page. Unpleasantly, we did so
```

Comment:
I actually still do two spaces myself for code since it's typically viewed in a monospace font rather than proportional.